docs/README.md

```diff
@@ -125,52 +131,52 @@ Now, you are able to use all the blocks exported by the `disclosure-layout` app.
-| `Hide`    | `Block` | This block will be rendered when prompt to hide the content | `undefined`   |
+| `Show`    | `block` | Name of the block that will be rendered when prompt to show the content. | `undefined`   |
+| `Hide`    | `block` | Name of the block that will be rendered when prompt to hide the content. | `undefined`   |
 
-### `disclosure-layout-group` props
+#### `disclosure-layout-group` props
 
 | Prop Name    | Type   | Description                                                                                                                                                 | Default value |
 | ------------ | ------ | ----------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------- |
-| `maxVisible` | `enum` | Possible values `one` it will render only one child `disclosure-layout` content per group, or `many` to render any number of `disclosure-layout`'s content. | `one`         |
+| `maxVisible` | `enum` | Defines how many `disclosure-layout` blocks should be displayed at time. Possible values are: `one` (only one `disclosure-layout` block should have its content displayed at time) or `many` (different `disclosure-layout`'s contents can be displayed at time). | `one`  | 
 
-### `disclosure-trigger-group` props
+#### `disclosure-trigger-group` props
 
 | Prop Name    | Type     | Description                                                                                          | Default value |
 | ------------ | -------- | ---------------------------------------------------------------------------------------------------- | ------------- |
-| `Show`       | `Block`  | This block will be rendered when prompt to show the content                                          | `undefined`   |
-| `Hide`       | `Block`  | This block will be rendered when prompt to hide the content                                          | `undefined`   |
-| `as`         | `string` | The HTML tag that will be used.                                                                      | `button`      |
-| `children`   | `Block`  | This block will be rendered if no `show` or `hide` is set                                            | `undefined`   |
-| `blockClass` | `string` | Block container class. This prop’s set value functions as a block identifier for CSS customizations. | `undefined`   |
+| `Show`       | `block`  | Name of the block that will be rendered when prompt to show the content.            | `undefined`   |
+| `Hide`       | `block`  | Name of the block that will be rendered when prompt to hide the content.            | `undefined`   |
+| `as`         | `string` | HTML tag to be applied to the component when it is rendered on the UI.              | `button`      |
+| `children`   | `block`  | Name of the block that will be rendered in case no blocks are declared in the `show` or `hide` props.           | `undefined`   |
```

Comment:
```suggestion
| `children`   | `block`  | Name of the block that will be rendered in case no blocks are declared in the `Show` or `Hide` props.           | `undefined`   |
```